roles/vouch/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+#
+# Manages the vouch job within Nomad
+# https://github.com/vouch/vouch-proxy
+#
+- name: Check if Vouch configuration has been loaded in Consul
```

Comment:
Does it make more sense to prompt for input at this point and manually build the config? My gut reaction is for this to not be super interactive. I definitely do not want all of this in variables.

Reply:
I feel like if you can check automatically that is better than asking, because the person being asked may not be super well aware of what's happening 🤷

Reply:
My point is that the current design expects a user to load the key into Consul manually and then rerun the playbook.

What I _could_ do is prompt for all of the inputs that need to be in the Consul key and then insert it into Consul automatically.

Reply:
I think the prompting route will be faster/more consistent, would do that